rfc-020-use-rollbar-to-monitor-rails-errors.md

```diff
@@ -0,0 +1,26 @@
+# Use Rollbar to monitor application errors for Rails projects
+
+## Summary
+
+Use Rollbar as the service to monitor and manage application errors for Rails
+ projects.
+
+## Problem
+
+We need to use a tool that collects errors that happen across our live
+ environments.
+We need a tool that can notify the team of new or important errors.
+We need a way to record and share the outputs of our investigations and any
+ plans to tackle the problem, more investigation, intended fixes etc.
+We need a way to group common issues.
```

Comment:
Did you want these to be separate paragraphs? If so, they need a blank line in between them.